### ISSUE TITLE
Add more status badges to Readme.md template

### ISSUE THREAD

**Issue by szymonkups** [ckeditor/ckeditor5-dev-env#6]
The `README.md` file template contains `devDependencies` status badge, we can add there also `dependencies` and `npm package` badges.

**Comment by Reinmar** [comment]
I think that the template is fine now. We don't have normal dependencies so far in nearly all packages anyway. We have only devDeps. And we don't have those packages on npm still, so the npm badge doesn't work as well.

We'll need to work on this, but later on.

**Comment by oskarwrobel** [comment]
Url for `devDependencied` is incorrect.

Now is:
```
https://david-dm.org/{{GitHubRepositoryPath}}#info=devDependencies
```

Should be:
```
https://david-dm.org/{{GitHubRepositoryPath}}?type=dev
```

`#info=devDependencies` => `?type=dev`

**Comment by Reinmar** [comment]
They changed this recently. We need to upgrade it everywhere.

**Comment by Reinmar** [comment]
I pushed a fix for @oskarwrobel's comments in #26.

**Comment by Reinmar** [comment]
WIll be fixed in https://github.com/ckeditor/ckeditor5-dev-env/issues/28.
